### PR TITLE
move env.sh to .env

### PR DESCRIPTION
**Pull request by durandom**
I have no idea what this `env.sh` would do.

`.env` is a more standard way of doing things.
Also updated the README

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+LADI_MODEL='map.sav' # path to SOM map to test against
+LADI_INDEX='logstash-' # name of Elasticsearch index to pull logs from
+LADI_TARGET_INDEX='mcliffor_test_ingest-' # name of index the anoamliy data will be pushed back to in Elasticsearch
+LADI_TIME_SPAN=60 # number of seconds from now that you would like to query Elasticsearch for
+LADI_MAX_ENTRIES=35000 # limits the number of entries returned from Elasticsearch for the inference query
+LADI_SERVICE='journal' # name of Elasticsearch service to be monitored
+LADI_THRESHOLD=1.5 # float tunes the anomaly threshold based on the max value of the training set
+
+LOCAL_DATA_FILE=validation_data/Hadoop_2k.json
+LOCAL_RESULTS_FILE=/tmp/local-results.txt
```
Comment (zmhassan) [inline .env.example]: @MichaelClifford Are we still using this file? I think you set some of these env vars with safe defaults

Comment (MichaelClifford) [reply .env.example]: @zmhassan, env.sh was a script used early on to manage and quickly load all of the needed environment variables. But,  AFAIU @vpavlin moved all of these into `config.py` or the appropriate storage class. So, I do not think this file is being used any more.

Comment (vpavlin) [reply .env.example]: It is definitely not used by OpenShift, but what @durandom is proposing IIUC is to use it in Travis

Comment (zmhassan) [reply .env.example]: @durandom what do you think. Should we delete this file?

Comment (durandom) [reply .env.example]: what file are you all talking about? 
`LOCAL_RESULTS_FILE` ? 
or `env.sh`

The former seems to be still used.
And `config.py` is still using env variables.

**Comment by durandom** [comment]
@zmhassan merge this one too?

**Comment by zmhassan** [comment]
Yup. This looks good. I'm going to merge.

**Pull request merged**